localstack/services/providers.py

```diff
@@ -184,7 +184,7 @@ def kms():
     from localstack.services.kms.provider import KmsProvider
 
     provider = KmsProvider()
-    return Service("kms", listener=AwsApiListener("kms", MotoFallbackDispatcher(provider)))
```

Comment:
As discussed briefly in a conversation - I understand that it would be nice to have some backup plan if this PR introduces some issues. But keeping two providers at once is going to be an effort. Also we might want to have a more general mechanism for reverting things to the last working setup. So something like a way for our customers to downgrade our version to the previous satisfactory would be a better way to do this.

Reply:
As discussed, as this implementation is essentially lift-and-shift (with a couple of enhancements added along the way), there is limited value in keeping a separate provider - especially as we have introduced additional AWS validated tests, as well as several fixes based on running the TF KMS tests. 👍